cpp/src/replace/replace.cu

```diff
@@ -47,22 +48,35 @@ namespace{ //anonymous
    * @returns
    */
   /* ----------------------------------------------------------------------------*/
-  template <class T>
+  template <class T,
+            bool is_col_valid, bool is_new_valid>
```

Comment:
We're adopting the term "nullable" to mean that a column has a valid mask. "is_valid" is ambiguous.

```suggestion
            bool col_is_nullable, bool new_is_nullable>
```